packages/gatsby-source-contentful/src/__tests__/gatsby-node.js

```diff
@@ -408,12 +415,16 @@ describe(`gatsby-node`, () => {
 
     const removedBlogEntry =
       startersBlogFixture.removeBlogPost.currentSyncData.deletedEntries[0]
+    const normalizedType = removedBlogEntry.sys.type.startsWith(`Deleted`)
+      ? removedBlogEntry.sys.type.substring(`Deleted`.length)
+      : removedBlogEntry.sys.type
```

Comment:
We have this logic here in tests but not in `deleteContentfulNode` function (in gatsby-node). It doesn't look right to me.

Does Contentful actually add `Deleted` prefix to types in `deletedEntries`? If so, we should account for this in the code as well, if not then we should probably just change fixtures instead of doing this.

Reply:
Yeah, it does - I've just added this logic into deleteContentfulNode as well. Thanks